README.md

```diff
@@ -1,33 +1,37 @@
 <img src="https://www.gdgalgiers.com/static/gdg_algiers-86a26a90b5a8da9cdd3127750215f051.png" alt="GDG LOGO" >
 <br>
 
-# templateJs
+# TemplateJs
 
+TemplateJs is an open source discord.js based bot template that will help you build a Discord bot in a quick and elegant way. It's Ultimate, efficient, easy-to-use and simple-to-setup Command handler for Discord bots, You just have to tweak few settings, add your commands, and you will be ready to go!
 
-## What is it ? 
-it's a template for discord bots with the discordJs library 
+## Requirements
+* ```git``` command line ([Windows](https://git-scm.com/download/win) | [Linux](https://git-scm.com/download/linux) | [MacOS](https://git-scm.com/download/mac)) installed
+* ```node``` [Version 16.x](https://nodejs.org/en/) or newer
+* [guild id](https://poshbot.readthedocs.io/en/latest/guides/backends/setup-discord-backend/#find-your-guild-id-server-id) of the server, [the client id and the token](https://poshbot.readthedocs.io/en/latest/guides/backends/setup-discord-backend/#prerequisites) of the bot
 
-## how to use it ?
+## How to use it ?
 
-* First you have to colone this repo 
+* Make sure that the '[applications.commands](https://discord.com/developers/applications/)' scope is checked in the scopes section of the OAuth2 settings for your bot
+
+* Clone this repo 
  ```git
  git clone https://github.com/GDGAlgiers/discord.js-bot-template
   ```
-* then you have to put the token of your bot , and the guild id and the client id of your discord server , int the config.js file 
-
+* Put the token and the client id of your bot, the guild id of your discord server, into the config.js file 
 
  ```
  {
-	"clientId": "your client id ",
+	"clientId": "your client id",
 	"guildId": "your guild id",
-	"token": "your bot's tokent"
+	"token": "your bot's token"
 }
   ```
-* install the dependencies
+* Install the dependencies
 ```npm
 npm install 
   ```
-* finaly , run the bot.js file 
+* Run the bot.js file 
 ```node
 node src/bot.js 
-  ```
\ No newline at end of file
+  ```
```

Comment:
I think we can add a section of warning of hiding the config.json file from pushing the token.

Reply:
Done, hope that's what you mean